src/source/source_cache.js

```diff
@@ -475,6 +477,7 @@ class SourceCache extends Evented {
             clearTimeout(this._timers[id]);
             this._timers[id] = undefined;
         }
+        // TODO should we keep this??
         this._source.fire('data', { tile: tile, coord: tile.coord, dataType: 'tile' });
```

Comment:
@lucaswoj @anandthakker do you think we need to fire an event when a tile is removed now that `tiledata` is ⚰️? I feel like this isn't useful, but let me know if you disagree.